usaspending_api/search/v2/elasticsearch_helper.py

```diff
@@ -53,79 +62,92 @@ def search_transactions(filters, fields, sort, order, lower_limit, limit):
         'query': {
             'query_string': {
                 'query': keyword
-                }
-            },
+            }
+        },
         'sort': [{
             query_sort: {
                 'order': order}
-            }]
-        }
-
-    transaction_type = next((award_type_mapping[k] for k in
-                             filters['award_type_codes']
-                             if k in award_type_mapping), 'contracts')
-
-    index_name = '{}-{}'.format(TRANSACTIONS_INDEX_ROOT,
-                                transaction_type.lower().replace(' ', ''))
-    index_name = index_name[:-1]+'*'
-    try:
-        response = CLIENT.search(index=index_name, body=query)
-    except Exception:
-        logging.exception("There was an error connecting to the ElasticSearch instance.")
-        return None
-    total = response['hits']['total']
-    results = format_for_frontend(response['hits']['hits'])
-    return results, total, transaction_type
-
+        }]
+    }
 
-def get_total_results(keyword, index_name):
-    index_name = '{}-{}'.format(TRANSACTIONS_INDEX_ROOT,
-                                index_name.replace('_', ''))+'*'
+    for index, award_types in indices_to_award_types.items():
+        if sorted(award_types) == sorted(filters['award_type_codes']):
+            index_name = '{}-{}-*'.format(TRANSACTIONS_INDEX_ROOT, index)
+            transaction_type = index
+            break
+    else:
+        logger.exception('Bad/Missing Award Types requested')
```

Comment:
Can we please direct users where to get the award type code/ requirements

Reply:
We should. But that is outside the scope of this sprint